.github/workflows/release.yml

```diff
@@ -5,6 +5,7 @@ on:
     # It should be kept in sync with if-condition in jobs
     - cron: '0 6 * * MON' # Storage release
     - cron: '0 6 * * THU' # Proxy release
+    - cron: '0 6 * * TUE' # Proxy release
```

Comment:
Should be `Compute` instead of proxy?
Did they want such schedule or they want to do it on-demand and manually trigger the workflow?

Reply:
Good catch! Fixed, thanks!